template/flake.nix

```diff
@@ -0,0 +1,19 @@
+{
+  description = "virtual environments";
+
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, flake-utils, devshell }:
+    flake-utils.lib.eachDefaultSystem (system: {
+      devShell = let shell = import devshell { inherit system; }; in
+        shell.mkShell {
+          imports = [ (shell.importTOML ./devshell.toml) ];
+
+          commands = [{
+            package = devshell.defaultPackage.${system};
+            help = "Per project developer environments";
```

Comment:
this wouldn't work out of the box isn't it?

Reply:
It does work with `nix develop`. Probably want to use _flake-compat_ for `nix-shell`. I was gonna do this last night but got tired :smile_cat:

Reply:
I went ahead and moved devshell to the toml, while demonstrating how imports work from nix. Useful for declaring local devshell modules.